Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -263,6 +264,22 @@ namespace AZ::Reflection
                 StackEntry* nodeData = &m_stack.back();
                 nodeData->m_path = AZStd::move(path);
 
+                // If our parent node is disabled then we should inherit its Disabled attribute and make note of the inheritance
+                if (parentData.m_disableEditor)
+                {
+                    nodeData->m_disableEditor = true;
+                    nodeData->m_isAncestorDisabled = true;
```

Comment:
This seems like the m_isAncestorDisabled would be stale if the parent node is then enabled.